pandas/core/frame.py

```diff
@@ -4321,21 +4321,41 @@ def combiner(x, y, needs_i8_conversion=False):
     def update(self, other, join='left', overwrite=True, filter_func=None,
                raise_conflict=False):
         """
-        Modify DataFrame in place using non-NA values from passed
-        DataFrame. Aligns on indices
+        Modify in place using non-NA values from another DataFrame.
+
+        Aligns on indices. There is no return value.
 
         Parameters
         ----------
         other : DataFrame, or object coercible into a DataFrame
+            Index should be similar to one of the columns in this one. If a
```

Comment:
I don't really understand "Index should be similar to one of the columns in this one". What do you want to say here?